app/js/main.js

```diff
@@ -201,6 +211,37 @@ function setupMeteringDemo(subscriptions) {
     location.reload();
   });
 
+  // Determine whether there is a publisher based entitlement.
```

Comment:
Could there be a publisher entitlement that exists yet doesn't unlock the current article?

Reply:
Could you create a buganizer to track this?  I can't consider it right now

Reply:
I could just imagine the publisher might have multiple tiers of subscriptions, like "basic" and "premium". If that's true, then I think the comment should be updated to say "Determine whether there is a publisher entitlement that unlocks the article"

Reply:
It's a very good point and the comment update is a good idea for now but I will create a buganizer for us to better consider the right behavior for premium/basic type subscriptions

Reply:
Sounds good, thanks!